exporter/metric/option.go

```diff
@@ -59,6 +59,10 @@ type options struct {
 	// If it is set to zero then default value is used.
 	ReportingInterval time.Duration
 
+	// EnforceUTF8 causes all metric and resource label values to be converted to valid UTF-8.
+	// If it is unset then default value (true) is used.
+	EnforceUTF8 *bool
```

Comment:
As I said in the PR description:
> I added the flag in case it was preferable to fail rather than ingest modified values, but we could also unconditionally sanitize those strings instead.

Do you feel that the option is overkill?

Reply:
Lets start without the option if that is alright.  We can always add it later

Reply:
Done.